certification_organism/data/product_product.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="__export__.product_product_3_ec1455d9" model="product.product" context="{'studio': True}">
```

Comment:
Why do you need to create this? Can't you just remove the '{'create_product_product': False}' in the product_template data and they will be created?